core/src/player.rs

```diff
@@ -477,22 +477,30 @@ impl Player {
 
             let max_frames_per_tick = self.max_frames_per_tick();
             let mut frame = 0;
+            let mut last_time_offset = self.time_offset;
 
             while frame < max_frames_per_tick && self.frame_accumulator >= frame_time {
                 let timer = Instant::now();
                 self.run_frame();
-                let elapsed = timer.elapsed().as_millis() as f64;
+                let elapsed = timer.elapsed().as_millis();
+                if frame > 0 && elapsed == 0 {
```

Comment:
Why `frame > 0`?

Reply:
Removed, was leftover from the last approach